src/app/aircraft/add/index.njk

```diff
@@ -41,23 +41,23 @@
 
         <p class="govuk-body-l">{{__('add_aircraft_caption')}}</p>
 
-        <div class="govuk-form-group {{ m.form_group_class('craftReg', errors) }}">
+        <div class="{{ m.form_group_class('craftReg', errors) }}">
```

Comment:
Tweaks to the form_group_class macro means that the first "govuk-form-group" is redundant. This will need applying throughout the codebase.